src/pkg/services/m365/api/groups_test.go

```diff
@@ -179,6 +180,11 @@ func (suite *GroupsIntgSuite) TestGroups_GetByID() {
 			id:        groupID,
 			expectErr: assert.NoError,
 		},
+		{
+			name:      "valid email as identifier",
+			id:        groupsEmail,
+			expectErr: assert.NoError,
+		},
```

Comment:
Can you also add an invalid email identifier check, please?  Just a fabricated email like `a@b.c` should do.

Reply:
done, thank you